src/main/java/com/github/onsdigital/babbage/api/util/SearchUtils.java

```diff
@@ -73,64 +62,60 @@
 public class SearchUtils {
 
     private static final String DEPARTMENTS_INDEX = "departments";
-    private static final String ERRORS_KEY = "errors";
 
     /**
      * Performs search for requested search term against filtered content types and counts contents types.
      * Content results are serialised into json with key "result" and document counts are serialised as "counts".
      * <p>
      * Accepts extra searches to perform along with content search and document counts.
      *
+     * @param searchTerm
+     * @param queries
+     * @param searchDepartments
+     * @return
+     */
+    public static LinkedHashMap<String, SearchResult> search(String searchTerm, SearchQueries queries, boolean searchDepartments) throws IOException {
```

Comment:
Following Java conventions I think its better to have a single return statement in a method. 
```
LinkedHashMap<String, SearchResult> results  = null;
if (searchTerm != null) {
    ...
}
return results;
```
Also do we want to return null if the search term is empty? If so we can use `StringUtils.isNotEmpty(...)` in the check instead of `!== null`.

Reply:
Added comment to future task in Trello to check for this scenario where the searchTerm is "".